### DISCUSSION TITLE
Deafen the Bot - Hear me Out

### DISCUSSION THREAD

**Discussion by alexysarts** [jagrosh/MusicBot#1469] [Ideas]
I think regardless of whether it helps save bandwidth or not, there are still two good reasons why you should have the bot deafened.

1) Some people feel paranoid or unsafe whether that's valid or not, and they'll be more likely to join a vc with a bot if they can see the deafened icon next to it. So I'm asking you for this so that members can feel more comfortable joining the vc. 
2) When you have a line-up of bots that deafen, (all the other ones do), and yours is the only one without that deafen icon, it looks goofy, so it's also more aesthetically pleasing to do this. If you manage to find the time to do this, I appreciate it.

**Comment by jagrosh** [comment]
Duplicate of #519
